finagle-example/src/main/java/com/twitter/finagle/example/java/http/LoggingFilter.java

```diff
@@ -0,0 +1,30 @@
+package com.twitter.finagle.example.java.http;
+
+import com.twitter.finagle.Service;
+import com.twitter.finagle.SimpleFilter;
+import com.twitter.finagle.http.Request;
+import com.twitter.finagle.http.Response;
+import com.twitter.util.Duration;
+import com.twitter.util.Function;
+import com.twitter.util.Future;
+import com.twitter.util.Stopwatch$;
+
+/**
+ * A simple Finagle filter that logs the Request total time in milliseconds.
+ */
+public final class LoggingFilter extends SimpleFilter<Request, Response> {
+
+    @Override
+    public Future<Response> apply(Request req, Service<Request, Response> service) {
+        final Duration e = Stopwatch$.MODULE$.start().apply();
```

Comment:
bummer that we don't have a nicer java API for `Stopwatch`. i'll try to address that for the future.

that said, you should capture the `Elapsed` from calling `start()`. then call `apply()` inside the `map` function below.